README.md

```diff
@@ -125,6 +125,9 @@ There are two flavors of live ISO images:
 
 You can download them from the [openSUSE Build
 Service](https://download.opensuse.org/repositories/systemsmanagement:/Agama:/Devel/images/iso/).
+* Make sure to download the correct ISO file according to your system architecture(Example: You would need to choose {https://download.opensuse.org/repositories/systemsmanagement:/Agama:/Devel/images/iso/agama-live.x86_64-7.0.0-openSUSE-Build1.17.iso} if your processor is based on x64 architecture.
+* And do not forget to check out the ALP vs openSUSE difference mentioned above and download the correct file based on your requirements.
+* Inorder to launch Agama using virtual machine please refer https://github.com/openSUSE/agama/issues/932#issuecomment-1863007573.
```

Comment:
```suggestion
* In order to launch Agama using virtual machine please refer https://github.com/openSUSE/agama/issues/932#issuecomment-1863007573.
```

Probably is better to add a section about VirtualBox. Maybe @lslezak or @mvidner  can help with this.

Reply:
I'd rather extract the relevant text here. Linking to a comment in another long discussion is not nice.

Reply:
> I'd rather extract the relevant text here. Linking to a comment in another long discussion is not nice.

Sure, that's what I meant with "a section about".

Reply:
Not only that: the comment talks about running the ISO *for development purposes*. IMHO it does not belong to the README.

Reply:
Exactly what @imobachgs said. That comment is not about using the downloaded ISO. Is about using that ISO as an starting point for Agama development. A completely different topic that does not belong here.